resources/sagemaker-apps.go

```diff
@@ -0,0 +1,90 @@
+package resources
+
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/sagemaker"
+	"github.com/rebuy-de/aws-nuke/pkg/types"
+)
+
+type SageMakerApp struct {
+	svc             *sagemaker.SageMaker
+	domainId        *string
```

Comment:
```suggestion
	domainID        *string
```

This does not follow the Golang naming conventions. Abbreviations should have consistent case. See [Guidelines](https://github.com/golang/go/wiki/CodeReviewComments#initialisms).

I know that the AWS-SDK does not follow that rule and some other names slippend through with an inconsistent naming, but we need to move towards a consistent naming.